portality/templates/publisher/metadata.html

```diff
@@ -28,13 +28,20 @@ <h4 class="red">There is a problem with the submitted form.</h4>
                         <button type="submit" id="remove_{{subfield.short_name}}" name="remove_{{subfield.short_name}}" class="btn btn-danger remove_button">
                           <i class="glyphicon glyphicon-remove"></i>
                         </button>
+                        {% for error in subfield.errors %}
+                            <ul class="errors">
+                                <li>{{ subfield.errors.get(error)[0] }}</li>
+                            </ul>
+                        {% endfor %}
                         </div>
                     {% endfor %}
                     <input type="submit" class="btn btn-info" name="more_authors" value="Add More Authors">
                     {% if author_error %}
-                    <ul class="errors">
-                        <li>Please provide at least one author</li>
-                    </ul>
+                    <div class="error">
+                        <ul class="errors">
+                            <li>Please provide at least one author</li>
+                        </ul>
+                    </div>
```

Comment:
What's the necessity for this extra div with the same class as the enclosed ul?

Reply:
It is not needed here apparently. But it is still needed above for orcid id error cause without it the red colour is not inherited. This is not the most elegant way I guess but the simplest one I think.

Reply:
It doesn't seem to be causing the red colour to be inherited even so - on my screen the error came up in black text.